app/controllers/api/v1/event_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::EventController < ApiController
+  def index
+    render json: Event.all
```

Comment:
I'd suggest using ActiveModelSerializers to prepare the json objects: https://github.com/rails-api/active_model_serializers